.github/workflows/dev-docker-publish.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
       - name: Get release_version
         id: ver
-        run: echo "release_version=$(go list -f '{{.Version}}' -m github.com/th2-net/th2-box-template-go@master)" >> $GITHUB_OUTPUT
+        run: echo "release_version=$(go list -f '{{.Version}}' -m github.com/th2-net/th2-box-template-go@box-template)" >> $GITHUB_OUTPUT
```

Comment:
This code should be independent from branch, I mean developer should not remember to update this script when create new branch